packages/frontend/core/src/desktop/pages/invite/index.tsx

```diff
@@ -23,16 +26,45 @@ import { AppContainer } from '../../components/app-container';
  * only for web
  */
 const AcceptInvite = ({
+  inviteId,
   inviteInfo,
 }: {
+  inviteId: string;
   inviteInfo: GetInviteInfoQuery['getInviteInfo'];
 }) => {
   const { jumpToPage } = useNavigateHelper();
+  const graphqlService = useService(GraphQLService);
+  const navigateHelper = useNavigateHelper();
+  const [error, setError] = useState<UserFriendlyError | null>(null);
 
   const openWorkspace = useCallback(() => {
     jumpToPage(inviteInfo.workspace.id, 'all', RouteLogic.REPLACE);
   }, [inviteInfo.workspace.id, jumpToPage]);
 
+  useEffect(() => {
```

Comment:
use effect will run twice?
maybe move this to service

Reply:
I made some changes, could you please review it again?